source/material_model/rheology/diffusion_creep.cc

```diff
@@ -176,6 +213,35 @@ namespace aspect
                            "Units: \\si{\\meter\\cubed\\per\\mole}.");
         prm.declare_entry ("Grain size", "1e-3", Patterns::Double (0.),
                            "Units: \\si{\\meter}.");
+	prm.declare_entry ("Use water fugacity for diffusion creep", "false",
+			   Patterns::Bool(),
+			   "Whether or not to include water fugacity in the diffusion creep rheology. Setting this to false "
+			   "gives the default diffusion creep rheology. If set to true, a compositional field named 'water' "
+			   "must exist, and the material model will query the water content at each mesh point and include "
+			   "that in the calculation for the viscosty. Wet rheological parameters must also be specified.");
+	prm.declare_entry ("Wet prefactors for diffusion creep", "3e-24",
+			   Patterns::Anything(),
+			   "List of wet viscosity prefactors, $A^{wet}$, for background material and compositional fields, "
+			   "for a total of N+1 values, where N is the number of compositional fields. "
+			   "If only one value is given, then all use the same value. "
+			   "Units: \\si{\\per\\pascal\\meter}$^{m_{\\text{diffusion}}}$\\si{\\per\\second}.");
+	prm.declare_entry ("Wet activation energies for diffusion creep", "375e3",
+			   Patterns::Anything(),
+			   "List of wet activation energies, $E_a^{wet}, for background material and compositional fields, "
+			   "for a total of N+1 values, where N is the number of compositional fields. "
+			   "If only one value is given, then all use the same value. "
+			   "Units: \\si{\\joule\\per\\mole}.");
+	prm.declare_entry ("Wet activation volumes for diffusion creep", "23e-6",
+                           Patterns::Anything(),
+                           "List of wet activation volumes, $V_a^{wet}$, for background material and compositional fields, "
+                           "for a total of N+1 values, where N is the number of compositional fields. "
+                           "If only one value is given, then all use the same value. "
+                           "Units: \\si{\\meter\\cubed\\per\\mole}.");
+	prm.declare_entry ("Water fugacity exponents for diffusion creep", "1.0",
```

Comment:
If you do change the sign of this exponent in the equations above, the default value should be -1, rather than 1.